### PR TITLE
AB#9231 Dont allow upload of executables

### PR DESCRIPTION
**Pull request by aurelio-aot**

### CODE REVIEW

File: applications/Unity.GrantManager/src/Unity.GrantManager.Web/appsettings.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "SecretAccessKey": "",
     "ApplicationS3Folder": "Unity/Application",
     "AssessmentS3Folder": "Unity/Adjudication",
-    "DisallowedFileTypes": "[ \"application/x-msdownload\" ]",
+    "DisallowedFileTypes": "[ \"exe\",\"sh\",\"ksh\",\"bat\",\"cmd\" ]",
```
Comment (AndreGAot) [inline applications/Unity.GrantManager/src/Unity.GrantManager.Web/appsettings.json:42]: have we tested this in the docker env? (linux, maybe also setting the openshift values in test to test as this was the error last time with these disallowed file types)

Comment (aurelio-aot) [reply applications/Unity.GrantManager/src/Unity.GrantManager.Web/appsettings.json:42]: > have we tested this in the docker env? (linux, maybe also setting the openshift values in test to test as this was the error last time with these disallowed file types)

yup, me and @DarylTodosichuk tested it, the escape character \ should be removed when the value is set in the openshift environment as S3__DisallowedFileTypes.

Comment (AndreGAot) [reply applications/Unity.GrantManager/src/Unity.GrantManager.Web/appsettings.json:42]: thanks - resolving

**Comment by AndreGAot** [comment]
I don't see it, but have we enforced at the API level these disallowed file types? Should be enforced in the UI and API level

**Comment by aurelio-aot** [comment]
> I don't see it, but have we enforced at the API level these disallowed file types? Should be enforced in the UI and API level

hi @AndreGAot I have now pushed the codes for 1) logic for disallowing invalid file types in the API level and 2) unit test.

**Review by AndreGAot — APPROVED** [review]
Looks good

**Pull request merged**